### ISSUE TITLE
Bible reference overlay in fullscreen mode could be draggable

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#310]

**Comment by rcmosher** [comment]
I assume this is referring to the little "Toast" message for the current verse that shows up when scrolling. I find it unhelpful, especially when in a split screen. Say I've got TSK open in the top window and the Bible in the bottom window, while scrolling the bottom pane to skim the text. My thumb will be covering part of the top of the bottom pane to scroll, with the verse reference at about 1/3 up the bottom pane. This leaves a very small window to actually read in. Maybe 3 lines.

Personally I find the reference completely unnecessary. But at least it should have its gravity set so it appears in the bottom right (or left for Hebrew and other RTL languages) to make it less prominent.

**Comment by ericjding** [comment]
I don't typically use a split screen. I like the Bible reference overlay, and find it helpful. I think if it were in the bottom right, then it would be less useful to me since then I would not be able to see it while scrolling because my right hand would be covering it as I scroll. I suppose this is another feature that needs to balance the preferences and usage patterns of different users...

**Comment by rcmosher** [comment]
Good point. Would keeping it centered, but moving it to the bottom of the screen work for you, @ericjding ? That should work for me. Of course an on/off switch would be enough for me.

**Comment by ericjding** [comment]
Yes, I think moving it toward the bottom would be fine if it's centered horizontally on the screen (though visually, I personally think it looks nicer to have a little distance from the bottom). Adding another option is, I think, overkill in this case; in that sense, I respect @mjdenham in his efforts to minimize configurability -- too many settings options leads to another kind of UI clutter.
